### PR TITLE
feat: Add device service files and make command options

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/developer-scripts/blob/master/.github/Contributing.md.

## What is the current behavior?

Commented out Device Service are removed when the new compose files are generated
The source compose files have the docker registry name in the file name, but the registry is now an environment variable so could be something different.

## What is the new behavior?

Make commands now have options for adding each device service to the generated compose file.
source compose file names have be refactored to simplify and remove the docker registry name.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: releases/nightly-build/compose-files/source/Makefile [hunk 1]
```diff
@@ -109,18 +131,25 @@ ui-down:
...
 	KONG_UBUNTU= \
 	docker-compose -p edgex \
-		-f docker-compose-nexus-base.yml \
-		-f docker-compose-nexus-add-device-services.yml \
-		-f docker-compose-nexus-add-mqtt.yml \
-		-f docker-compose-nexus-add-security.yml \
+		-f docker-compose-base.yml \
+		-f add-device-bacnet.yml \
+		-f add-device-modbus.yml \
+		-f add-device-modbus.yml \
```
Comment (jim-wang-intel) [inline releases/nightly-build/compose-files/source/Makefile:145]:
```suggestion
		-f add-device-grove.yml \
```

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]

File: releases/nightly-build/compose-files/source/Makefile (hunk 1 above)
Comment (lenny-intel) [reply releases/nightly-build/compose-files/source/Makefile:145]: Dang it! Good catch!

Comment (lenny-intel) [reply releases/nightly-build/compose-files/source/Makefile:145]: fixed

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Comment by lenny-intel** [comment]
@cherrycl, this PR with minor changes to the committed compose files used by TAF is failing the new smoke tests. Can you please review the failures here and determine why it is failing? I have no clue why :-(
https://jenkins.edgexfoundry.org/blue/rest/organizations/jenkins/pipelines/edgexfoundry/pipelines/edgex-taf-pipelines/branches/smoke-test/runs/17/log/?start=0

BTW, I ran get-compose.sh with and w/o security and the resulting compose files looks good. 
Thanks!

**Review by jim-wang-intel — DISMISSED** [review]

**Review by jpwhitemn — DISMISSED** [review]
Nuice!  Worked well.  I tested with each of the DS additions and only the Grove had issues (appropriate for x86).

**Comment by cherrycl** [comment]
@lenny-intel The error occurred by [edgex-taf PR #198](https://github.com/edgexfoundry/edgex-taf/pull/198), not about the compose files. For this PR, you can ignore the failure. I will fix the error on edgex-taf repo.

**Comment by cherrycl** [comment]
@lenny-intel The merged [edgex-taf PR #208](https://github.com/edgexfoundry/edgex-taf/pull/208) fixed the failed smoke tests.

**Comment by lenny-intel** [comment]
> The merged [edgex-taf PR #208](https://github.com/edgexfoundry/edgex-taf/pull/208) fixed the failed smoke tests.

@cherrycl , Thank you for the very quick response to this issue!!

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**